utils/frame/remote-externalities/src/lib.rs

```diff
@@ -235,8 +154,10 @@ impl Default for SnapshotConfig {
 
 /// Builder for remote-externalities.
 pub struct Builder<B: BlockT> {
-	/// Pallets to inject their prefix into the externalities.
+	/// certain custom key-pairs to be injected into the externalities.
 	inject: Vec<KeyPair>,
+	/// Certain prefixes the be injected as well. The raw prefix must be given.
```

Comment:
```suggestion
	/// Storage entry key prefixes to be included in the runtime state mock. The raw prefix must be given.
```
Would it be correct to say "runtime state mock" == "externalities"?

Reply:
correct